Maestro/subscriptions.json

```diff
@@ -1144,14 +1144,12 @@
         "vsoSourceBranch": "nightly"
       }
     },
+    // Reviewers: Should these changes be merged into release/3.1 branches instead?
```

Comment:
@anurse @Pilchie your thoughts on ignoring public release/3.0 branches?

Reply:
Given that 2.2 is out of support, and 3.0 is a current release with a newer LTS:

* 2.2 should not be getting any more fixes, because we're not shipping it anymore.
* 3.0 should be getting fixes from 2.1, but we'll need to consider whether they are functional or security fixes.  3.0 should only be getting security fixes at this point.

Maybe we should be flowing directly from 2.1 -> 3.1, and hand merging security fixes into 3.0?

Reply:
@Pilchie @dougbu I didn't think that 2.1 and 3.1 had similar enough repo layouts to allow for easy merges between them?

Reply:
Let's chat about branch merging offline.

Reply:
Actually, we do expect one more public change in 3.0 (SameSite fix) so it would be best to keep the syncing to 3.0.

Let's add chatting about branch merging to our weekly build/leads sync agenda though.

Reply:
Oof, missed some of the earlier discussion while composing my responses ;).

I'm OK with hand-merging. We have one 3.0-approved public change (https://github.com/dotnet/aspnetcore/pull/17953 , because it's a regression from 3.0.0 introduced in 3.0.1) but we can hand-merge it.